opm/simulators/wells/WellInterface_impl.hpp

```diff
@@ -1189,6 +1189,26 @@ namespace Opm
             }
         }
     }
+
+    template <typename TypeTag>
+    void
+    WellInterface<TypeTag>::
+    setPrevSurfaceRates(WellState& well_state, 
+                            const WellState& prev_well_state) const
+    {
+        const int np = this->number_of_phases_;
```

Comment:
There is no for loop needed, it will work with something like, 

`ws.prev_surface_rates = prev_well_state.well(this->index_of_well_).surface_rates;`

and 

`ws.prev_surface_rates = ws.surface_rates;`

Reply:
Thanks! Done